src/BigQueryHelper.php

```diff
@@ -405,7 +409,22 @@ private function _runQuery($sql, $async = false, $legacySql = true, $extraOpts =
     {
       return $client->startQuery($jobConfig);
     }
-    return $client->runQuery($jobConfig, $queryResultsOptions);
+    $result = $client->runQuery($jobConfig, $queryResultsOptions);
+
+    // Check whether the query has completed
+    if(($info = $result->info()) && empty($info['jobComplete']))
```

Comment:
don't use assignments in an `if`

```suggestion
    $info = $result->info();
    if($info && empty($info['jobComplete']))
```